### DISCUSSION TITLE
Will groups information be added in lock after 2.20.0?

### DISCUSSION THREAD

**Discussion by monchin** [pdm-project/pdm#3251] [Q&A]
Hello, I found that after updating pdm to 2.20.0.post1, `pdm lock` removes `groups` information in lock files.

For me, groups information is important, because I need to collect licenses for packages in some certain groups. I also found that pep 751 also support groups (although it's a draft now). I wonder why groups information is removed from 2.20.0, and will it be added later?

**Comment by frostming** [comment]
> Hello, I found that after updating pdm to 2.20.0.post1, `pdm lock` removes `groups` information in lock files.

This is not true. Check the `strategy` value

**Reply by monchin** [reply]
I see, strategy must include "inherit_metadata", or there is no groups information for each package in lock.
